tests/jquery.flot-relativeTimeAxis.Test.js

```diff
@@ -113,4 +113,32 @@ describe('A Flot chart with relative time axes', function () {
         expect(plot.getAxes().xaxis.ticks[0].label).toEqual('00:00:00.000');
     });
 
+    it('works with multiple axis', function () {
```

Comment:
multiple axes test should have multiple data series, each assigned to a different axis